tests/unit/TableTest.php

```diff
@@ -95,13 +82,7 @@ public function testUpdateRecord()
 
     public function testDeleteRecord()
     {
-        $table = $this->database->createTable(
-            new TableSchema('test-delete-record', [
-                new Column('id', ColumnType::STRING(), true),
-                new Column('first_name', ColumnType::STRING()),
-                new Column('last_name', ColumnType::STRING())
-            ])
-        );
+        $table = $this->createTestTable('test-delete-record',false);
```

Comment:
In reference to one of my comments below, remove the false value
and instead, add an age field with a value to the Record declaration.